API.ts

```diff
@@ -0,0 +1,60 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import { FirstRunValues } from "./customTypes";
+import { LoginFormValues } from "./Screens/Login";
+
+export const storeData = async (key, value) => {
```

Comment:
you could probably add comments to each of these API functions to explain what they are for